save-backend/src/test/kotlin/org/cqfn/save/backend/DownloadFilesTest.kt

```diff
@@ -14,7 +15,7 @@ import org.springframework.web.reactive.function.BodyInserters
 import kotlin.io.path.ExperimentalPathApi
 
 @WebFluxTest
-class DownloadFilesTest {
+class DownloadFilesTest : DatabaseTestBase() {
```

Comment:
This test doesn't use testcotnainers, does it?

Reply:
No, it doesn't